scripts/qa_crowbarsetup.sh

```diff
@@ -3260,6 +3260,8 @@ function addfloatingip
     nova floating-ip-create | tee floating-ip-create.out
     floatingip=$(perl -ne "if(/\d+\.\d+\.\d+\.\d+/){print \$&}" floating-ip-create.out)
     nova add-floating-ip "$instanceid" "$floatingip"
+    # flush arp tables in case this floating ip was recently used for something else
+    arp -d "$floatingip"
```

Comment:
@rossella that only updates the localhost's arp table, right? that doesn't fix anything. in my case the l3 router is on the *other* network node, and there the arp entry is indeed correct. however on *my* network node it is not updated and hence points to some outdated garbage.